items/equip-1hand/item576_Beheader.xml

```diff
@@ -9,7 +9,12 @@
   <item id="576" image="equipment/weapon/sword-beheader.png" name="Beheader" description="The only sword capable of beheading a Terranite." effect="Double Damage +50% / No bow / 0 Int" criticalattack="-25" hit="-20" type="equip-1hand" attack="999" attack-action="attack" attack-range="2" weight="2000" sellProtected="true">
     <sprite gender="unisex">equipment/weapons/sword-short.xml|#48488c,48488e,48488f,695c21,5f5fbe,827229,6e6edb,9493a4,a4adff,bab8c2,bcbbc6,e3eaff</sprite>
     <sprite gender="female">equipment/weapons/sword-short-female.xml|#48488c,48488e,48488f,695c21,5f5fbe,827229,6e6edb,9493a4,a4adff,bab8c2,bcbbc6,e3eaff</sprite>
-    <sound event="strike">weapons/knives/knife-miss1.ogg</sound>
-    <sound event="hit">weapons/knives/knife-hit1.ogg</sound>
+    <sound event="strike">weapons/other/staff/staff-miss1.ogg</sound>
+    <sound event="strike">weapons/other/staff/staff-miss2.ogg</sound>
+    <sound event="strike">weapons/other/staff/staff-miss3.ogg</sound>
+    <sound event="hit">weapons/other/scythe/scythe-hit1.ogg</sound>
+    <sound event="hit">weapons/other/scythe/scythe-hit2.ogg</sound>
+    <sound event="hit">weapons/other/scythe/scythe-hit3.ogg</sound>
+
```

Comment:
Yes, a blank line is enough to TMWA CIs go haywire doomsday mode and uhh attempt to behead you.